src/lms7002m/LMS7002M.cpp

```diff
@@ -2215,10 +2193,15 @@ int LMS7002M::WriteGFIRCoefficients(TRXDir dir, uint8_t gfirIndex, const float_t
         uint8_t bankRow = i % bankLength;
         addrs[i] = startAddr + (bank * 8) + bankRow;
         addrs[i] += 24 * (bank / 5);
+
         if (i < coefCount)
+        {
             words[i] = coef[i] * 32767;
```

Comment:
worth adding a warning if the coef is not withing -1:1, as it would result in integer overflow.